### ISSUE TITLE
Flitter 1.17 push has no effect on IOS

### ISSUE THREAD

**Issue by liwangyong** [MaikuB/flutter_local_notifications#636]
**IMPORTANT**
If you are getting a build error on Android that mentions AndroidX incompatibilities, before reporting an issue, please make sure look at the error information carefully as there should be a URL that provides information on migrating your application to AndroidX.

Also note that the plugin has an example application that should cover all of the available functionality. Please check the example application and see if the functionality works before logging an issue. You'll need to clone the repository to run it

**Describe the bug**

**To Reproduce**
If applicable, steps to reproduce the behavior:

**Expected behavior**
If applicable, clear and concise description of what you expected to happen.

**Sample code to reproduce the problem**
Ideally, include a link to a repository that contains code that can reproduce the problem. Suggestion is to fork this repository to modify the example app. Note that without a sample, it'll be may be more difficult to reproduce the problem and/or investigate further

**Comment by MaikuB** [comment]
When filing a bug report then please fill in the details that's in the template instead of keeping the content that is there. Currently your post doesn't provide much info. Furthermore I've run this on 1.17 and haven't run into issues. There's been no similar reports about this either. There's an example app in the repository that can be used after downloading/cloning the repository

**Comment by silverlovesl** [comment]
I've same problem like issue owner. I copied example/main.dart source code to my project.
It works fine for Android, but nothing happened on IOS
But I run your example project, it worked on iOS. @MaikuB 

My flutter env follow 
[✓] Flutter (Channel stable, v1.17.1, on Mac OS X 10.15.4 19E287, locale en-JP)
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
[✓] Android Studio (version 3.6)
[✓] VS Code (version 1.45.1)
[✓] Connected device (3 available)

**Comment by MaikuB** [comment]
Do not expect it to work if you just copy the code. This is stated in the readme and comments within the example app's code as configuration is needed. If you're going to proceed with copying then you need to do all of the platform-specific setup. Otherwise you could download/clone the repository where this has already been done

**Comment by silverlovesl** [comment]
> Do not expect it to work if you just copy the code. This is stated in the readme and comments within the example app's code as configuration is needed. If you're going to proceed with copying then you need to do all of the platform-specific setup. Otherwise you could download/clone the repository where this has already been done

Thank @MaikuB  for answering
I've fixed problem after adding 
```
if((UserDefaults.standard.object(forKey: "Notification") == nil)){
        UIApplication.shared.cancelAllLocalNotifications()
        UserDefaults.standard.bool(forKey: "Notification")
    }
    
    if #available(iOS 10.0, *) {
        UNUserNotificationCenter.current().delegate = self;
    }
```
to AppDelegate

**Comment by liwangyong** [comment]
> > 如果只是复制代码，不要指望它能起作用。在需要配置的情况下，示例应用程序代码中的自述文件和注释中对此进行了说明。如果要进行复制，则需要进行所有特定于平台的设置。否则，您可以下载/克隆已经完成的存储库
> 
> 感谢@MaikuB 的回答
> 我添加后已解决问题
> 
> ```
> if((UserDefaults.standard.object(forKey: "Notification") == nil)){
>         UIApplication.shared.cancelAllLocalNotifications()
>         UserDefaults.standard.bool(forKey: "Notification")
>     }
>     
>     if #available(iOS 10.0, *) {
>         UNUserNotificationCenter.current().delegate = self;
>     }
> ```
> 
> 到AppDelegate

me too

**Comment by liwangyong** [comment]
> 提交错误报告时，请填写模板中的详细信息，而不要保留其中的内容。目前，您的帖子没有提供太多信息。此外，我已经在1.17上运行过此功能，也没有遇到任何问题。也没有类似的报告。
> 
> 存储库中有一个示例应用程序，可在下载/克隆存储库后使用。如果您还没有这样做，我建议您先检查一下

Sorry, I didn't add the relevant configuration
